packages/experiments/src/components/SelectedItemsList/SelectedItemsList.types.ts

```diff
@@ -51,15 +50,15 @@ export interface ISelectedItemsListProps<T> extends React.ClassAttributes<any> {
   /**
    * The selection
    */
-  selection?: Selection;
+  focusedItemIndices?: number[];
   /**
    * Gets the copy text that will be set in the item.
    */
   getItemCopyText?: (items: T[]) => string;
   /**
    * Function that specifies how the selected item will appear.
    */
-  onRenderItem: React.ComponentType<ISelectedItemProps<T>>;
+  onRenderItem?: React.ComponentType<ISelectedItemProps<T>>;
```

Comment:
There is no default renderer, shouldn't this still be required?

Reply:
Yes true, but there is a issue with Typescript, I will fix it later.